compiler/rustc_mir/src/borrow_check/diagnostics/mod.rs

```diff
@@ -388,7 +388,7 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                     //
                     // We know the field exists so it's safe to call operator[] and `unwrap` here.
                     let (&var_id, _) =
```

Comment:
Wait are you saying that it should return ty::CapturedPlace or that it is currently? Currently, this would be a hir id, since we are getting the index that returns a key-value pair and are taking the key not the value.

Reply:
ah I thought had removed the comment here 😅